src/components/Form/Collection/Collection.jsx

```diff
@@ -242,6 +242,22 @@ export default class Collection extends ValidationElement {
       )
     }
 
+    // Create the 'byline' which is used to separate items as well as the bottom toggle action
+    const bylineBottom = (item, index) => {
+      if (this.state.items.length < 2) {
+        return ''
+      }
+
+      return (
+        <div className="byline bottom">
+          <a href="javascript:;;" className="toggle" onClick={this.toggle.bind(this, index)}>
```

Comment:
It's a good question.

I believe the `href` attribute is optional according to written specifications, but in HTML 4.x it is considered a **hyperlink** if the attribute is present or an **anchor** if the attribute is not present. In HTML 5+ it is always considered a **hyperlink** but if the attribute is missing then it becomes just a placeholder for one.

In some scanners/linters/etc. the rules get confusing and I think the inconsistency between the two major versions is part of it. The inconsistency is also found in browsers as well. For example in some browsers if the `href` is omitted then the mouse cursor or screen reader does not recognize it as a hyperlink.

In reality we could possible not include it and it would work as intended on most newer browsers. If anyone has any input then we definitely would love to hear them!

References:
 [1]: https://www.w3.org/MarkUp/html-spec/html-spec_7.html
 [2]: http://www.w3schools.com/tags/tag_a.asp